targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/api/lp_ticker.c

```diff
@@ -139,6 +139,10 @@ void lp_ticker_set_interrupt(timestamp_t timestamp)
 
     /* Checking if LPTRM can handle this sleep */
     delta_ticks = USEC_TO_COUNT(delta_us, CLOCK_GetFreq(kCLOCK_Er32kClk));
+    if (delta_ticks == 0)
```

Comment:
it would be safer with brackets

Reply:
I have update the PR to include the brackets